src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.PlatformNotSupported.cs

```diff
@@ -16,444 +16,589 @@ public static class Sse
+        ///   MOVSS xmm, xmm
         /// </summary>
         public static Vector128<float> MoveScalar(Vector128<float> upper, Vector128<float> value) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_movehl_ps (__m128 a,  __m128 b)
+        /// __m128 _mm_movehl_ps (__m128 a,  __m128 b);
+        ///   MOVHLPS xmm, xmm
         /// </summary>
-        public static Vector128<float> MoveHighToLow(Vector128<float> left,  Vector128<float> right) { throw new PlatformNotSupportedException(); }
+        public static Vector128<float> MoveHighToLow(Vector128<float> left, Vector128<float> right) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_movelh_ps (__m128 a,  __m128 b)
+        /// __m128 _mm_movelh_ps (__m128 a,  __m128 b);
+        ///   MOVLHPS xmm, xmm
         /// </summary>
-        public static Vector128<float> MoveLowToHigh(Vector128<float> left,  Vector128<float> right) { throw new PlatformNotSupportedException(); }
+        public static Vector128<float> MoveLowToHigh(Vector128<float> left, Vector128<float> right) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// int _mm_movemask_ps (__m128 a)
+        /// int _mm_movemask_ps (__m128 a);
+        ///   MOVMSKPS reg, xmm
         /// </summary>
         public static int MoveMask(Vector128<float> value) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_mul_ps (__m128 a, __m128 b)
+        /// __m128 _mm_mul_ps (__m128 a, __m128 b);
+        ///   MULPS xmm, xmm/m128
         /// </summary>
-        public static Vector128<float> Multiply(Vector128<float> left,  Vector128<float> right) { throw new PlatformNotSupportedException(); }
+        public static Vector128<float> Multiply(Vector128<float> left, Vector128<float> right) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_mul_ss (__m128 a, __m128 b)
+        /// __m128 _mm_mul_ss (__m128 a, __m128 b);
+        ///   MULPS xmm, xmm/m32
         /// </summary>
         public static Vector128<float> MultiplyScalar(Vector128<float> left, Vector128<float> right) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_or_ps (__m128 a,  __m128 b)
+        /// void _mm_prefetch(char* p, int i);
+        ///   PREFETCHT0 m8
         /// </summary>
-        public static Vector128<float> Or(Vector128<float> left,  Vector128<float> right) { throw new PlatformNotSupportedException(); }
+        public static unsafe void Prefetch0(byte* address) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
-        /// __m128 _mm_rcp_ps (__m128 a)
+        /// void _mm_prefetch(char* p, int i);
+        ///   PREFETCHT1 m8
+        /// </summary>
+        public static unsafe void Prefetch1(byte* address) { throw new PlatformNotSupportedException(); }
```

Comment:
Should this rather have `void*` argument?

Reply:
Probably, will update.

Reply:
Fixed.